src/components/Config/Config.tsx

```diff
@@ -120,68 +147,105 @@ function Config() {
       </Typography.Title>
       {completed && (
         <Completed>
-          <p>You can now start saving on gas fees! Click one of the links below to proceed.</p>
+          <p>You can now start scheduling transactions! </p>
         </Completed>
       )}
       {!completed && (
         <>
-          <Checkboxes>
-            <CheckboxSection>
-              <Checkbox
-                checked={gasPriceAware}
-                disabled={submitted}
-                onChange={(e) => setGasPriceAware(e.target.checked)}
-              >
-                <Typography.Paragraph className="subtitle">Gas price aware</Typography.Paragraph>
-              </Checkbox>
-              <p>
-                Once the network gas price falls below the gas cost specified in the transaction, it will be broadcast
-                to the network.
-              </p>
-            </CheckboxSection>
-            <CheckboxSection>
-              <Checkbox checked={draft} disabled={submitted} onChange={(e) => setDraft(e.target.checked)}>
-                <Typography.Paragraph className="subtitle">Draft mode (Advanced)</Typography.Paragraph>
-              </Checkbox>
-              <p>
-                No transaction will be broadcast to the Ethereum network until you go to the Transaction list and
-                specify additional conditions.
-              </p>
-            </CheckboxSection>
-          </Checkboxes>
-          <div>
-            <Typography.Paragraph className="subtitle">Estimated Confirmation Time</Typography.Paragraph>
-            <p>
-              Our algorithms analyze historical gas prices in real time to best propose a gas price for you. Longer wait
-              times generally correspond to cheaper gas prices. Since we cannot control the Ethereum network, this is
-              NOT a guaranteed confirmation time.
-            </p>
-            <SliderContainer>
-              <Slider
-                marks={sliderMarks}
-                step={1}
-                value={confirmationTime}
-                min={0}
-                max={3}
-                disabled={submitted}
-                tooltipVisible={false}
-                onChange={setConfirmationTime}
+          <Radio.Group
+            defaultValue={Network.None}
+            onChange={(e) => setnetwork(e.target.value)}
+            size="large"
+            className="title"
+          >
+            <Radio.Button value={Network.Ethereum} className="radiobuttons">
+              <img alt="eth-network-icon" src="/assets/eth.svg" width="32" height="32" className="network-icon" />
+              Ethereum
+            </Radio.Button>
+            <Radio.Button value={Network.Arbitrum}>
+              <img alt="arb-network-icon" src="/assets/arbitrum.svg" width="32" height="32" className="network-icon" />
+              Arbitrum
+            </Radio.Button>
+          </Radio.Group>
+
+          {network === Network.Ethereum && (
+            <>
```

Comment:
No need for this anymore

Reply:
if i remove this the landing page will show eth. connection settings. To have it like on the image, i am keeping it
![image](https://user-images.githubusercontent.com/34207598/162963640-d8e0aecd-37ba-4701-972e-1b48cbf66af7.png)

Reply:
I don't get it. How is `<>` affecting the layout?